# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: 496ec0c5a60f
# Full Hash: 496ec0c5a60fdb86b233d036dd4747c477fc33ca
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-10-31 09:53:09
# Regressor Bug: 1581637
# File Overlap Count: 1
# Description:
#   Bug 1581637 - Part 8 - Add Http3Session/Http3Stream. r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46652
# ==============================================================================

diff -r 63b7f1ff1714 -r 496ec0c5a60f netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Wed Oct 30 20:54:21 2019 +0000
+++ b/netwerk/protocol/http/moz.build	Wed Oct 30 21:09:09 2019 +0000
@@ -77,6 +77,8 @@
     'Http2Push.cpp',
     'Http2Session.cpp',
     'Http2Stream.cpp',
+    'Http3Session.cpp',
+    'Http3Stream.cpp',
     'HttpAuthUtils.cpp',
     'HttpBackgroundChannelChild.cpp',
     'HttpBackgroundChannelParent.cpp',
@@ -134,6 +136,7 @@
     '/netwerk/base',
     '/netwerk/cookie',
     '/netwerk/ipc',
+    '/netwerk/socket/neqo_glue',
     '/netwerk/url-classifier',
 ]
 